src/Samples/Sample.AspNetCore/Controllers/CheckOutController.cs

```diff
@@ -104,7 +104,7 @@ public async Task<SwedbankPay.Sdk.Payments.Card.Payment> CreateCardPayment()
                                                                                    new PayeeInfo(this.payeeInfoOptions.PayeeId,
                                                                                                  this.payeeInfoOptions.PayeeReference));
 
-                SwedbankPay.Sdk.Payments.Card.Payment cardPayment = await this.swedbankPayClient.Payment.CreateCreditCardPayment(cardRequest);
+                SwedbankPay.Sdk.Payments.Card.Payment cardPayment = await this.swedbankPayClient.CreateCreditCardPayment(cardRequest);
```

Comment:
Can we have this read like the following? 

```suggestion
                SwedbankPay.Sdk.CardPayments.CardPayment cardPayment = await this.swedbankPayClient.CardPayments.Create(cardRequest);
```

Reply:
I've ripped out that part of the `SwedbankPayClient`. If you want to use one singular part of the api at a time you could just use `IPaymentsClient` instead of the `ISwedbankPayClient`.